ocs_ci/ocs/cluster.py

```diff
@@ -3823,3 +3823,51 @@ def get_age_of_cluster_in_days():
     seconds_per_day = 24 * 60 * 60
     time_diff_in_days = time_difference_in_sec / seconds_per_day
     return math.ceil(time_diff_in_days)
+
+
+def get_ceph_pool_list():
```

Comment:
we have a similar function here https://github.com/red-hat-storage/ocs-ci/blob/3d8e5d8f68210023671c50e10f0ba3b81bca3035/ocs_ci/ocs/rados_utils.py#L124

Reply:
I will check

Reply:
I will remove these functions to reduce the PR size and include them in the next PR.